extensions/amp-nexxtv-player/0.1/amp-nexxtv-player.js

```diff
@@ -95,12 +95,13 @@ class AmpNexxtvPlayer extends AMP.BaseElement {
         'The data-client attribute is required for <amp-nexxtv-player> %s',
         this.element);
 
-    const start = this.element.getAttribute('data-seek-to') || '0';
+    const delay = this.element.getAttribute('data-delay') || '0';
```

Comment:
Renaming this will be a breaking change and not allowed since component has already launched. 
If you really care about renaming, deprecation is one approach where you support both and deprecate the old one.

Reply:
ok, I'll find a way to handle this.